src/components/AssetIconWithBadge.tsx

```diff
@@ -22,7 +22,7 @@ const StatusStyle = { svg: { transform: 'scale(0.8)' } }
 type AssetIconWithBadgeProps = {
   size?: AvatarProps['size']
   assetId?: AssetId
-  transfersByType: Record<TransferType, Transfer[]>
+  transfersByType?: Record<TransferType, Transfer[]>
```

Comment:
Tweak this interface to make it more generic - `AssetIconWithBadgeProps` does not require `transfersByType`.